README.md

```diff
@@ -1,14 +1,19 @@
-#### To get started
+## To get started
 
+```sh
+yarn
+# or npm i
+yarn start
+# or npm start
```

Comment:
a bit opinionated, but *really* opinonated would be not including the `npm i` line 😄 

I could switch their order (and have yarn be the comment)